src/useCases/getAllUsers.ts

```diff
@@ -13,7 +13,7 @@ const getAllUsers = async (
   if (visibleForces !== "") {
     const forces = visibleForces.split(",")
     /* eslint-disable @typescript-eslint/naming-convention */
-    const forceWhere = forces.map((code) => `visible_forces like '%${code}%'`).join(" OR ")
+    const forceWhere = forces.map((code) => `visible_forces ~ '(\\y|,)${code}(\\y|,)'`).join(" OR ")
```

Comment:
More of a question than a suggestion: why are we checking for word boundary _or_ comma? My understanding is that a comma will match a word boundary. I would be interested in knowing whether everything still works if you use just `\y` rather than `(\y|,)`.

But also, in the spirit of saving time - if you've tested this and it works, then just go for it 👍